### PR TITLE
Micro: Return error if tensor allocation fails.

### PR DESCRIPTION
**Pull request by jvistrand**

### CODE REVIEW

**Comment by gbaned** [comment]
@jvistrand Can you please resolve conflicts? Thanks!

**Comment by jvistrand** [comment]
Fixed. Also corrected some indentation on master.

File: tensorflow/lite/micro/examples/network_tester/network_tester_test.cc [hunk 1]
```diff
@@ -70,7 +70,12 @@ TF_LITE_MICRO_TEST(TestInvoke) {
 
   tflite::MicroInterpreter interpreter(model, resolver, tensor_arena,
                                        TENSOR_ARENA_SIZE, error_reporter);
-  interpreter.AllocateTensors();
+
+  TfLiteStatus allocate_status = interpreter.AllocateTensors();
+  if (allocate_status != kTfLiteOk) {
+    error_reporter->Report("alloc failed\n");
```
Comment (wangtz) [inline tensorflow/lite/micro/examples/network_tester/network_tester_test.cc]: Prefer allocation or tensor allocation failed.

**Review by wangtz — DISMISSED** [review]
Looks good. Thx.

File: tensorflow/lite/micro/examples/network_tester/network_tester_test.cc (hunk 1 above)
Comment (jvistrand) [reply tensorflow/lite/micro/examples/network_tester/network_tester_test.cc]: Changed it to "Tensor allocation failed"

**Review by wangtz — DISMISSED** [review]

**Comment by gbaned** [comment]
@jvistrand Can you please resolve conflicts? Thanks!

**Review by freddan80 — APPROVED** [review]

**Review by wangtz — APPROVED** [review]

**Pull request merged**